EuPathDB/src/main/resources/assets/admin/admin-ui.js

```diff
@@ -0,0 +1,332 @@
+
+/********************************************
+ *** Bootstrap various pages
+ *******************************************/
+
+var globalState = {
+  loadingRequests: 0,
+  subscriptionMeta: null,
+  groupMeta: null
+};
+
+$(function() {
+
+  // discover URL and id query param if present
+  let pathArray = window.location.pathname.split("/");
+  let page = pathArray[pathArray.length - 1];
+  let query = window.location.search.substr(1).split("&").filter(p => p.startsWith("id"));
+  let id = query.length > 0 ? query[0].substr(3) : undefined;
```

Comment:
If the following works, it should be more robust?

```suggestion
  let params = new URLSearchParams(window.location.search);
  let id = params.get("id") || undefined;
```

Reply:
(we won't have any admin users on ancient browsers that don't support this)